Uzoamaka126/My-Auto-Mart-Challenge
# [Feature #165933350] Implement AutoMart Landing Page

#### What does this PR do?
Implement the Landing Page

#### Description of Task to be completed?
- [x] Design and implementation of AutoMart Landing Page interface

#### How should this be manually tested?
* Clone https://github.com/Uzoamaka126/My-Auto-Mart-Challenge
* Visit `/index.html` on your browser
* You should see the image below

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
#165933350

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/46677256/57864988-b726a700-77f4-11e9-9612-4094a9de3dbb.png)


#### Questions:
None
